### PR TITLE
HoC 2024 - Add skinny banner to /teach and /administrators pages

### PR DESCRIPTION
**Pull request by kelbyhawn**
Adds an Hour of Code 2024 skinny banner to https://code.org/teach and https://code.org/administrators. These will only show when `hoc_mode` is set to `soon-hoc`.

## Links 
Jira ticket: [ACQ-2303](https://codedotorg.atlassian.net/browse/ACQ-2303)
Figma mock: [here](https://www.figma.com/design/emvQNdidNnPvoU1NucF7Re/Hour-of-Code-2024?node-id=2710-554&t=7pohnyU7jtKFg631-1)

## Testing story
Tested locally to make sure it only shows for `soon-hoc`.

----

## Teach page
teach 

## Administrators page
administrators

## Responsive

https://github.com/user-attachments/assets/47c89503-085e-4930-827b-45f78cbb56d1

## RTL
localhost code org_3000_teach(Code org - 1366x768)

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/administrators.haml [hunk 1]
```diff
@@ -10,8 +10,7 @@ theme: responsive_full_width
   .wrapper
     .text-wrapper.col-50
       %h1=hoc_s(:admins_page_top_heading)
-      %p.heading-xs
-        =hoc_s(:admins_page_top_subhead)
+      %p=hoc_s(:admins_page_top_subhead)
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/administrators.haml:13]: I updated this to match the subhead styling on the /teach page.

File: pegasus/sites.v3/code.org/public/administrators.haml [hunk 2]
```diff
@@ -20,11 +19,18 @@ theme: responsive_full_width
...
   = view :curriculum_launch_skinny_banner
 - else
   = view :section_divider_line
 
+-# Show Hour of Code banner if hoc_mode is set to "soon-hoc"
+- if DCDO.get('hoc_mode', 'soon-hoc')
+  = view :hoc_skinny_banner
+- else
+  = view :section_divider_line
+
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/public/administrators.haml]: Does this mean in the case of `curriculum-launch-2024` being false and `hoc_mode` not being `soon-hoc` we'd have two back to back divider lines? Or will we never have those two settings in that configuration?

File: pegasus/sites.v3/code.org/public/teach.haml [hunk 3]
```diff
@@ -21,6 +21,9 @@ theme: responsive_full_width
 
 = view :curriculum_launch_skinny_banner
 
+-# Show Hour of Code 2024 banner if hoc_mode is set to "soon-hoc"
+= view :hoc_skinny_banner
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/public/teach.haml:26]: Similar question here, would it be possible for us to end up with two back-to-back skinny banners?

**Review by TurnerRiley — APPROVED** [review]
Overall looks great! I do have a couple questions on different flag configs and how that pans out

File: pegasus/sites.v3/code.org/public/administrators.haml (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/administrators.haml]: Technically we could get back to back banners, but I don't think these ddco flags will ever be set at the same time. BUT you're right, the divider line will show when only one banner is hidden so I just removed it and added a light gray background to the section below 😅

File: pegasus/sites.v3/code.org/public/teach.haml (hunk 3 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/teach.haml:26]: Same answer above, technically possible but unlikely to happen 🙌

File: pegasus/sites.v3/code.org/public/administrators.haml [hunk 4]
```diff
@@ -20,12 +19,13 @@ theme: responsive_full_width
       %img.rounded-corners.box-shadow-primary{src: "/images/admins-page-top.png", alt: "", style: "position: relative; width: 100%;"}
     .clear
 
-- if !!DCDO.get('curriculum-launch-2024', false)
-  = view :curriculum_launch_skinny_banner
-- else
-  = view :section_divider_line
+-# Show Curriculum Launch banner if dcdo is true
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/administrators.haml:22]: I'm going to update the dcdo flag used here to  be more evergreen so will update this comment to include the dcdo flag when I do.

Comment (TurnerRiley) [reply pegasus/sites.v3/code.org/public/administrators.haml:22]: Sounds good!

**Review by TurnerRiley — APPROVED** [review]

**Pull request merged**